package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Assists in the creation of ATNA audit trail messages for IHE profiles.",
   "main": "index.js",
   "scripts": {
-    "test": "tap --cov test.js"
+    "pretest": "npm run gen:rng",
+    "test": "tap --cov test.js",
+    "gen:rng": "sed 's/#.*//' dicom.rnc > temp.rnc && trang temp.rnc dicom.rng && rm temp.rnc"
```

Comment:
One option could be to just take out the comments in the rnc (seeing as we're modifying it anyway)? Would help with platform-independence.